nikola/plugins/command/check.py

```diff
@@ -262,7 +252,7 @@ def analyze(self, task, find_sources=False, check_remote=False):
                         self.logger.warn("Broken link in {0}: {1}".format(filename, target))
                         if find_sources:
                             self.logger.warn("Possible sources:")
-                            self.logger.warn("\n".join(_call_nikola_list(self.l, self.site, ["--deps", task])))
+                            self.logger.warn("\n".join(_call_nikola_list(self.site)))
```

Comment:
I'd prefer JSON output; that's easy to parse and one can also easily add more info without breaking existing clients.